sense_energy/sense_link.py

```diff
@@ -49,42 +61,48 @@ def datagram_received(self, data, addr):
                     # Build response
                     response = plug.generate_response()
                     json_resp = orjson.dumps(response)
-                    encrypted_resp = tp_link_encrypt(json_resp.decode('utf-8'))
+                    encrypted_resp = tp_link_encrypt(json_resp.decode("utf-8"))
                     # Strip leading 4 bytes for...some reason
                     encrypted_resp = encrypted_resp[4:]
 
                     # Allow disabling response
                     if self.should_respond:
                         # Send response
-                        logging.debug(f"Sending response: {response}")
+                        logging.debug("Sending response: %s", response)
                         self.transport.sendto(encrypted_resp, addr)
                     else:
                         # Do not send response, but log for debugging
-                        logging.debug(f"SENSE_RESPONSE disabled, response content: {response}")
+                        _LOGGER.debug("SENSE_RESPONSE disabled, response content: %s", response)
```

Comment:
This was converting the json payload to a string and than throwing it away if debug logging was not enabled